### DISCUSSION TITLE
Opacity of info text

### DISCUSSION THREAD

**Discussion by jaid33** [kalkih/mini-media-player#823] [General]
Hi, 
with full-cover and white album art it is pretty hard to read the info text.
I was playing around with card-mod to change text shadow and the overlay at the bottom, but one tricky thing is that the info-text has some opacity (mmp-info-opacity: 0.75). 
If I change this value manually in the browser it seems to be much better, but I can't find a way to change it permanently.
That's my approach, everything except the --mmp-info-opacity is working.

`card_mod: `
`  style: | `
`    :host { `
`      --mmp-info-opacity: 0;`
`      text-shadow: 0px 0px 1px rgba(0, 0, 0, 0.6);`
`      --mini-media-player-media-cover-info-color: white; `
`      --mini-media-player-overlay-color-stop: 25%;`
`      --mini-media-player-overlay-color: rgba(0, 0, 0, 0.5)`
`    }`

**Comment by zookh0** [comment]
I got this working if there's artwork being displayed by using:
```
ha-card {
  --mmp-info-opacity: 1;
}
```
But I still haven't been able to get it to work when the artwork is set to none.
